lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/persistentStorage.py

```diff
@@ -1,5 +1,5 @@
 try:
-    from PySide6.QtCore import QSettings
+    from PySide6.QtCore import QSettings  # type: ignore
 except:
     from PySide2.QtCore import QSettings  # type: ignore
 
```

Comment:
I discovered that this writes files in a random folder. I think we will need to have some DCC-specific callback to save the data in DCC-specific locations. (In another PR)